features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,709 @@
+                            "passwordRecovery.form.fields.passwordRecoveryOtpUseLowercase.hint") as ReactNode }
+                </Hint>
+                <Field.Checkbox
+                    ariaLabel="passwordRecoveryOtpUseNumeric"
+                    name="passwordRecoveryOtpUseNumeric"
+                    label= { t("extensions:manage.serverConfigurations.accountRecovery." +
+                        "passwordRecovery.form.fields.passwordRecoveryOtpUseNumeric.label") }
+                    required={ false }
+                    readOnly={ readOnly }
+                    width={ 10 }
+                    // Disabling the last enabled option is not allowed
+                    disabled={ !isConnectorEnabled
+                        || (!isUpperCaseEnabled && !isLowerCaseEnabled && isNumericEnabled) }
+                    listen={ (value: boolean) => setIsNumericEnabled(value) }
+                    data-testid={ `${ testId }-sms-otp-numeric` }
+                    data-componentid={ `${ testId }-sms-otp-numeric` }
+                />
+                <Hint>
+                    { t("extensions:manage.serverConfigurations.accountRecovery." +
+                            "passwordRecovery.form.fields.passwordRecoveryOtpUseNumeric.hint") as ReactNode }
+                </Hint>
+                <Field.Input
+                    ariaLabel="otpLength"
+                    inputType="number"
+                    name="smsOtpLength"
+                    min={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .SMS_OTP_CODE_LENGTH_MIN_VALUE
+                    }
+                    max={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .SMS_OTP_CODE_LENGTH_MAX_VALUE
+                    }
+                    label= { t("extensions:manage.serverConfigurations.accountRecovery." +
+                                    "passwordRecovery.form.fields.passwordRecoveryOtpLength.label") }
+                    placeholder="SMS OTP Length"
+                    required={ false }
+                    maxLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.SMS_OTP_CODE_LENGTH_MAX_LENGTH
+                    }
+                    minLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.SMS_OTP_CODE_LENGTH_MIN_LENGTH
+                    }
+                    readOnly={ readOnly }
+                    width={ 10 }
+                    labelPosition="right"
+                    disabled={ !isConnectorEnabled }
+                    data-testid={ `${ testId }-otp-length` }
+                    data-componentid={ `${ testId }-otp-length` }
+                >
+                    <input/>
```

Comment:
Is this empty input intentional?

Reply:
yes this is used as the main input field